graphql/user_resolver.go

```diff
@@ -37,11 +38,21 @@ func (r *userResolver) Patches(ctx context.Context, obj *restModel.APIDBUser, pa
 	return &Patches{Patches: apiPatches, FilteredPatchCount: count}, nil
 }
 
+// Permissions is the resolver for the permissions field.
 func (r *userResolver) Permissions(ctx context.Context, obj *restModel.APIDBUser) (*Permissions, error) {
 	return &Permissions{UserID: utility.FromStringPtr(obj.UserID)}, nil
 }
 
+// Target is the resolver for the target field.
+func (r *subscriberInputResolver) Target(ctx context.Context, obj *restModel.APISubscriber, data string) error {
+	panic(fmt.Errorf("not implemented: Target - target"))
```

Comment:
Is this supposed to panic?

Reply:
This was auto generated. @evergreen-ci/evg-ui how do you want to handle it?

Reply:
I'd strongly want to not panic fwiw

Reply:
This only gets generated if there's a mismatch between the go struct and the graphql schema is this possible there's a new target field on the subscription input field

Reply:
[subscription input schema](https://github.com/evergreen-ci/evergreen/blob/main/graphql/schema/types/user.graphql#L67)
[APISubscriber](https://github.com/evergreen-ci/evergreen/blob/main/rest/model/subscriber.go#L16)

the type for `target` are mismatched, but neither seem like new changes

Reply:
I guess the only questions I have here are, can this code ever be hit? If not, is it still necessary to keep as part of the schema, or can it be removed?

Reply:
We have this ticket open to address it: https://jira.mongodb.org/browse/EVG-18186